jury/core.py

```diff
@@ -199,3 +200,27 @@ def evaluate(
     ) -> Dict[str, float]:
         """Returns __call__() method. For backward compatibility."""
         return self.__call__(predictions=predictions, references=references, reduce_fn=reduce_fn)
+
+
+if __name__ == "__main__":
```

Comment:
Was this left intentionally for example usage?

Reply:
Nope, it's redundant, will delete.